### ISSUE TITLE
Failed to extract tool 'Tor'

### ISSUE THREAD

**Issue by Bil97** [joelverhagen/TorSharp#81]
"Failed to extract tool 'Tor'. Verify that the .zip at path 'C:\Users\Bil\Imports\tor-win64-0.4.5.9.zip' is valid." when running multiple instanses of Torsharp

**Comment by joelverhagen** [comment]
Perhaps try assigning a different value to `TorSharpSettings.ZippedToolsDirectory` and `TorSharpSettings.ExtractedToolsDirectory` per instance? I haven't tested this particular scenario but this may mitigate the problem.

**Comment by Bil97** [comment]
I have to create multiple folders for each instance. This rises the disk usage to 100%. It would be great if one folder was used as the source from which new instances are created. Others wise its now working great.

**Comment by joelverhagen** [comment]
I've added some guidance on how to run multiple instances of Tor in parallel. Give it a try. If you're still having issues, feel free to open a new issue.
https://github.com/joelverhagen/TorSharp#can-i-run-multiple-instances-in-parallel
